src/embedded-checkout/embed-checkout.ts

```diff
@@ -0,0 +1,43 @@
+import EmbeddedCheckout from './embedded-checkout';
+import { EmbeddedCheckoutEventMap } from './embedded-checkout-events';
+import EmbeddedCheckoutOptions from './embedded-checkout-options';
+import { EmbeddedContentEvent } from './iframe-content/embedded-content-events';
+import IframeEventListener from './iframe-event-listener';
+import IframeEventPoster from './iframe-event-poster';
+import LoadingIndicator from './loading-indicator';
+import parseOrigin from './parse-origin';
+import ResizableIframeCreator from './resizable-iframe-creator';
+
+/**
+ * Embed the checkout form in an iframe.
+ *
+ * Once the iframe is embedded, it will automatically resize according to the
+ * size of the checkout form. It will also notify the parent window when certain
+ * events have occurred. i.e.: when the form is loaded and ready to be used.
+ *
+ * ```js
+ * embedCheckout({
+ *     url: 'https://checkout/url',
+ *     container: 'container-id',
```

Comment:
`containerId`

Reply:
Thanks. This issue has been resolved with 2e52c9d.